dask_cuda/utils.py

```diff
@@ -764,3 +764,18 @@ def get_rmm_memory_resource_stack(mr) -> list:
         if isinstance(mr, rmm.mr.StatisticsResourceAdaptor):
             return mr.allocation_counts["current_bytes"]
     return None
+
```

Comment:
Linter is failing because another blank line is missing here. I suggest installing `pre-commit` and running `pre-commit install` in each repo before committing, which will automatically fix such things for you. You can still run `pre-commit run -a` to fix linting issues and committing those changes.

Reply:
Yeah - I usually have pre-commit installed, so just forgot I had a new environment.